lizmap/modules/lizmap/lib/Request/WFSRequest.php

```diff
@@ -288,6 +300,10 @@ public function getfeature()
         return $this->getfeatureQgis();
     }
 
+    /**
+     * https://en.wikipedia.org/wiki/Web_Feature_Service#Static_Interfaces
+     * Queries Qgis Server for getFeature.
```

Comment:
decription should be above the `@see` (that you should add for the url).

Fix this for all comments you added